plugin/src/test/java/com/epam/grandhackathon/deployment/sphere/plugin/utils/EnvVarsResolverTest.java

```diff
@@ -0,0 +1,77 @@
+package com.epam.grandhackathon.deployment.sphere.plugin.utils;
+
+import java.util.logging.Level;
+
+import jenkins.model.Jenkins;
+import lombok.extern.java.Log;
+import hudson.EnvVars;
+import hudson.model.AbstractBuild;
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.TaskListener;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
+import hudson.util.LogTaskListener;
+
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+
+import com.epam.grandhackathon.deployment.sphere.plugin.action.DynamicVariablesStorageAction;
+import com.epam.grandhackathon.deployment.sphere.plugin.metadata.Constants;
+import com.epam.grandhackathon.deployment.sphere.plugin.parameter.DeployMetaDataParameterValue;
+
+@RunWith(MockitoJUnitRunner.class)
+@Log
+public class EnvVarsResolverTest {
+	
+	@Mock
+	private AbstractBuild<?, ?> build;
+
+	@Mock
+	private TaskListener taskListener;
+	@InjectMocks
+	private EnvVarsResolver envVarsResolver;
+
+	@Rule
+    public JenkinsRule j = new JenkinsRule();
+	
+	private static final String TEST_ENV_KEY = "qa";
+	private static final String TEST_BUILD_VERSION = "0.8";
+	private static final String TEST_APP_NAME = "app1";
+	
+	private EnvVarsResolver resolver;
+
+	@Before
+	public void setUp() throws Exception {
+		FreeStyleProject project = j.createFreeStyleProject("Deployment test job");	
```

Comment:
is this set up specific for all tests? if no it's better to encapsulate within test method then extract if any